helm/hpcc/templates/roxie.yaml

```diff
@@ -175,7 +175,7 @@ kind: Service
 metadata:
   name: {{ $commonCtx.toponame | quote }}
   labels:
-    helmVersion: 8.8.0-closedown0
+{{ include "hpcc.addStandardLabels" (dict "root" $ "component" "topology-server" "name" $roxie.name "instance" $commonCtx.toponame) | indent 4 }}
```

Comment:
I'm not clear about "name" here, I thought it should be a fixed name of the application, with instance being the instance of that application, so here name would be "topology-server" (although as we discussed offline, it would be better if the naming convention made it distinct from the deployment name),

And, as with comments on/off line, re. component on things like PVC's and services, I think component be something like "service" here.

Reply:
components like PVCs, which are not necessarily dedicated to a single app, therefore not part of a particular component can be categorized into generic "storage", etc component.
This particular service seems to be dedicated to a particular app/component roxie/topology-server